rts/motoko-rts/src/types.rs

```diff
@@ -460,12 +465,16 @@ impl Blob {
         self.add(1) as *mut u8 // skip closure header
     }
 
-    pub unsafe fn len(self: *mut Self) -> Bytes<u32> {
+    pub unsafe fn payload_const(self: *const Self) -> *const u8 {
```

Comment:
Rust has no overloading, so I had to choose another name. What do you think of renaming `payload_addr` to `payload_mut`?

Reply:
No object, but maybe check if _addr is already used consistently elsewhere. Perhaps we should mandate hungarian notation in the backend ;->

Reply:
Hungarian notation is a tool to enforce context-freeness in naming. As a Hungarian, I disapprove of it generally :-)

I think `payload_addr` exists for arrays too, so that would need a change also. This is not at all urgent, so let it soak and decide next time this comes up.